src/QueryEditor/QueryOptions.tsx

```diff
@@ -177,7 +177,7 @@ export const QueryOptions = (props: Props): JSX.Element => {
         <InlineField
           label="Aggregation"
           labelWidth={labelWidth}
-          tooltip="Specify an aggregation, leave empty to query raw data"
+          tooltip="Specify an aggregation and an interval to fetch an aggregated measurement value for each interval, or remove both to fetch raw data"
```

Comment:
this is not correct,  an interval is optional.

```suggestion
          tooltip="Set an aggregation (and optionally an interval) to fetch grouped data, or leave empty for raw values"
```

Reply:
Fetch an aggregated value per interval (set no interval to apply full time period), or leave both empty for raw values